extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -24,11 +24,38 @@ limitations under the License.
 
 # amp-subscriptions
 
-## Introduction
+## Usage
 
 The `amp-subscriptions` extension implements subscription-style access/paywall rules.
 
-## Relationship to `amp-access`
+The solution comprises the following components:
```

Comment:
Could you elaborate on why this section is here? What do you mean by "components"? Because AMP is built using AMP's components this is a tricky word.

Reply:
It's meant to give a high-level overview of the various parts that make up the component. Maybe this is redundant with the "Page landing flow" section. WDYT?